scripts/run_tracker.py

```diff
@@ -172,6 +178,156 @@ def trigger_analysis(run, config):
+          " it does not contain the Header section !!".format(FCID)))
+        return False
+
+    if not samplesheet_dict["Data"]:
+        LOG.warn(("When trying to generate SampleSheet.csv for sample sheet {} I find out that "
+                  " it does not contain the Data section !!".format(FCID)))
+        return False
+
+
+    FCID_samplesheet_dest_dict = {}
+    FCID_samplesheet_dest_dict["Header"] = []
+    for header in samplesheet_dict["Header"]:
+        FCID_samplesheet_dest_dict["Header"].append(header)
+
+
+    FCID_samplesheet_dest_dict["Data"] = []
+    for data in samplesheet_dict["Data"]:
+        if data: #if not empty
+            new_data = data[0:6] + data[7:8] # remove index2 fuck Genologics
+            if data[0] != "Lane": #this is the header section
+                tmp = new_data[1]
+                new_data[1] = new_data[2] # make SampleID equal to SampleName
+                new_data[2] = tmp     # in this way the dir structure will be project/sample_name/*fastq
+        FCID_samplesheet_dest_dict["Data"].append(new_data) # keep also white lines
+
+    #I do not care if there are other [.*] sections
+    return FCID_samplesheet_dest_dict
+
+
+
+def dictToSampleSheet(samplesheet_dict, file_dest):
+    """ Writes samplesheet stroed in samplesheet in a form of hash table
+        into a destination file
+        ASSUMPTION: this is a Xten sample sheet, contains Header and Data sections
+        
+        :param dict samplesheet: samplesheet stored in a dict
+        :param str file_dest: destination file
+        
+    """
+    try:
+
+        with open(file_dest, 'wb') as csvfile:
+            writer = csv.writer(csvfile, delimiter=',')
+            writer.writerow(["[Header]"])
+            for header in samplesheet_dict["Header"]:
+                writer.writerow(header)
+            writer.writerow(["[Data]"])
+            for data in samplesheet_dict["Data"]:
+                writer.writerow(data)
+    except ValueError:
+        LOG.warning("Corrupt samplesheet %s, please fix it" % file_dest)
+        pass
+
+
+def sampleSheetToDict(samplesheet):
+    """ takes as input a samplesheet (Xten compatible) and stores all field in an hash table.
+        Samplesheet should look something like:
+            [Section1]
+            section1,raw,1
+            section2,raw,2
+            [Section2]
+            section2,raw,1
+            section2,raw,2
+            ...
+        the hash structure will look like
+            "Section1" --> [["section1", "raw", "1"],
+                            ["section1", "raw" , "2"]
+                           ]
+            "Section2" --> [["section2", "raw", "1"],
+                            ["section2", "raw" , "2"]
+                           ]
+    
+        :param str samplesheet: the sample sheet to be stored in the hash table
+    """
+    samplesheet_dict = {}
+    try:
+        section = ""
+        header = re.compile("^\[(.*)\]")
```

Comment:
it feels weird that you've to treat the headers this way, but well I don't know any other way, and if people hasn't complained I guess they don't know either ;-)